docs/home/howto/kubernetes/PERSISTENT_STORAGE.md

```diff
@@ -90,6 +90,25 @@ specification:
 
 #### Create disks for Rook/Ceph Cluster Storage - AWS
 
+To define additional empty disk resources for Rook/Ceph Cluster Storage on AWS, use `specification.disks.additional_disks`:
```

Comment:
Maybe here it will be good to write to which object you need to attach this configuration.